src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -89,9 +93,26 @@ object CassandraLauncher {
    * @throws akka.persistence.cassandra.testkit.CassandraLauncher.CleanFailedException if `clean`
    *   is `true` and removal of the directory fails
    */
-  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit = this.synchronized {
-    if (cassandraDaemon.isEmpty) {
+  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit =
+    start(cassandraDirectory, configResource, clean, port, fork = false)
```

Comment:
Should we deprecate this function? (and when we actually remove it add a default value of `false` to the parameter in the other variation?)